front/lib/api/assistant/configuration.ts

```diff
@@ -171,6 +173,13 @@ export async function getAgentConfigurations(
   const rawAgents = await AgentConfiguration.findAll({
     where: {
       workspaceId: owner.id,
+      ...(agentPrefix
+        ? {
+            name: {
+              [Op.iLike]: `${agentPrefix}%`,
```

Comment:
You might also want to filter in `getGlobalAgents` 🙏🏻 
Workspace agents are stored in database, and Global Agents defined by Dust are stored directly on the code, as they are built and retrieved from the user's workspace.